runs/services.sh

```diff
@@ -43,17 +43,9 @@ start() {
 
 
 	openwbDebugLog "MAIN" 1 "Starting smart home handler..."
-	local smartmq=$(<"$OPENWBBASEDIR/ramdisk/smartmq")
-	if (( smartmq == 0 ))
-	then
-		openwbDebugLog "MAIN" 2 "Using legacy smarthomehandler"
-		local smarthomehandler_enabled='smarthomehandler'
-		local smarthomehandler_disabled='smarthomemq'
-	else
-		openwbDebugLog "MAIN" 2 "Using smarthomemq"
-		local smarthomehandler_enabled='smarthomemq'
-		local smarthomehandler_disabled='smarthomehandler'
-	fi
+	openwbDebugLog "MAIN" 2 "Using smarthomemq"
```

Comment:
Die Zeile könnte weg, denn sie fügt der Logausgabe in der Zeile davor nichts hinzu.